### PR TITLE
Added reference to the 'Up for Grabs' label

### PR DESCRIPTION
**Pull request by csharpfritz**

### CODE REVIEW

File: NuGet.Docs/Contribute/Default.cshtml [hunk 1]
```diff
@@ -1,4 +1,13 @@
 ﻿<div class="article-boxes">
+
+    <article class="box links wide">
+	<header>
+	    <h1>NuGet Contributions</h1>
+        </header>
+	<p>NuGet is an open source project and welcomes contributions from anyone in the communiuty.  Your contributions will all be held to the same standard as those on the project team.</p>
+	<p>For a quick list of items that the NuGet team has identified for anyone in the community to resolve, check the <a href="https://github.com/NuGet/Home/issues?q=is%3Aopen+is%3Aissue+label%3A%22Up+for+Grabs%22">Up for Grabs</a> label in the issues list within the Home repository.</p>
```
Comment (yishaigalatzer) [inline NuGet.Docs/Contribute/Default.cshtml]: What if instead of anyone in the community we say:

has identified as good issues to look into

**Comment by csharpfritz** [comment]
Updated and rebased from feedback

**Comment by maartenba** [comment]
There's also an Up for grabs label in the gallery, but we have not updated the items that are in it for a while.

**Comment by csharpfritz** [comment]
I don't see the Up for Grabs label in the [NuGetGallery](https://github.com/NuGet/NuGetGallery/labels) repository... should it be there?

**Comment by yishaigalatzer** [comment]
Go ahead and add it

**Comment by maartenba** [comment]
Looks like a milestone https://github.com/NuGet/NuGetGallery/milestones/Up%20For%20Grabs

**Comment by csharpfritz** [comment]
Hmm..  should we convert the milestone to a label?



Jeff

**Comment by yishaigalatzer** [comment]
That works for me

**Comment by maartenba** [comment]
Yeah and perhaps triage what is really relevant - lots of older issues it seems

**Comment by harikmenon** [comment]
No longer valid for docs

**Pull request closed without merging**